etc/ipsec.conf

```diff
@@ -16,9 +16,9 @@ conn %default
 conn rw
     # http://wiki.loopop.net/doku.php?id=server:vpn:strongswanonopenvz
     # https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling
-    leftsubnet=0.0.0.0/0
+    leftsubnet=0.0.0.0/0,::/0
     # end ref
     leftfirewall=yes
     right=%any
-    rightsourceip=10.8.0.0/16
+    rightsourceip=10.8.0.0/16,2a00:1450:400c:c05::/64
```

Comment:
Oh, you are right, the *IPv4 mapped addresses* maps to the entire IPv4 internet. `fc00::/7` is the way to go.